.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug report
```

Comment:
Oh, one more comment. This bug report template and the feature request template below it are not mentioned anywhere in the PR description. Was this an accidental addition?